web/src/message_edit.js

```diff
@@ -867,11 +868,30 @@ export function save_inline_topic_edit($row) {
         return;
     }
 
+    const $message_header = $row.find(".message_header").expectOne();
+    const stream_id = Number.parseInt($message_header.attr("data-stream-id"), 10);
+    const stream_topics = stream_topic_history.get_recent_topic_names(stream_id);
+    if (stream_topics.includes(new_topic)) {
+        confirm_dialog.launch({
+            html_heading: $t_html({defaultMessage: "Merge with another topic?"}),
+            html_body: render_confirm_merge_topics_with_rename({
+                topic_name: new_topic,
+            }),
+            focus_submit_on_open: false,
```

Comment:
Why is this false?

Reply:
I feel like the default behavior probably isn't to confirm and go ahead, since people often don't realize they're doing this. I'd prefer the modal not have submit focused when open, but that's just a UX choice and if you want to suggest a reason to do it the other way (or have me open a design thread about it) we can chat about it!

Reply:
Yeah I'm not sure what's best, we can see what feedback we get on that detail (if any).